zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/FlowNodeInstanceProcessInstanceHandler.java

```diff
@@ -88,9 +88,11 @@ public void updateEntity(
     entity.setBpmnProcessId(recordValue.getBpmnProcessId());
     entity.setTenantId(tenantOrDefault(recordValue.getTenantId()));
 
-    // Parent tree path is intentionally not calculated here
-    entity.setTreePath(null);
-    entity.setLevel(0);
+    // Parent tree path is intentionally not calculated here, we set the value as if the instance is
+    // on the 1st level
+    // will be changed within https://github.com/camunda/camunda/issues/18378
+    entity.setTreePath(recordValue.getProcessInstanceKey() + "/" + record.getKey());
+    entity.setLevel(1);
```

Comment:
❌ please also update unit test to reflect this change `FlowNodeInstanceProcessInstanceHandlerTest .shouldUpdateEntityFromRecord` is now failing